maddr/mobilemoney_test.go

```diff
@@ -0,0 +1,52 @@
+package maddr
```

Comment:
let's use blackbox testing so we can see what the api surface looks like in the tests. `package maddr_test` oughta do it